gnd/src/main/java/com/google/android/gnd/GndApplication.java

```diff
@@ -23,6 +23,7 @@
 import androidx.work.Configuration;
 import androidx.work.WorkManager;
 import com.akaita.java.rxjava2debug.RxJava2Debug;
+import com.facebook.stetho.Stetho;
```

Comment:
Sorry to be a pain, but is it possible to split out the Stetho changes into a separate PR?

Just keeps the history slightly cleaner; if we need to revert our use of Stetho or our project persistence we can do so separately.